plugins/org.jkiss.dbeaver.erd.ui/src/org/jkiss/dbeaver/erd/ui/notations/ERDNotationDescriptor.java

```diff
@@ -22,11 +22,11 @@
 
 public class ERDNotationDescriptor extends AbstractDescriptor {
 
-    public static final String ERD_STYLE_NOTATION_EXT_ATTRIBUTE_ID = "id"; // $NON-NLS-N$
-    public static final String ERD_STYLE_NOTATION_EXT_ATTRIBUTE_NAME = "name"; // $NON-NLS-N$
-    public static final String ERD_STYLE_NOTATION_EXT_ATTRIBUTE_DESCRIPTION = "description"; // $NON-NLS-N$
-    public static final String ERD_STYLE_NOTATION_EXT_ATTRIBUTE_DEFAULT = "isDefault"; // $NON-NLS-N$
-    public static final String ERD_STYLE_NOTATION_EXT_ATTRIBUTE_NOTATION = "notation"; // $NON-NLS-N$
+    private static final String EXT_ATTRIBUTE_ID = "id"; // $NON-NLS-N$
+    private static final String EXT_ATTRIBUTE_NAME = "name"; // $NON-NLS-N$
+    private static final String EXT_ATTRIBUTE_DESCRIPTION = "description"; // $NON-NLS-N$
+    private static final String EXT_ATTRIBUTE_DEFAULT = "isDefault"; // $NON-NLS-N$
+    private static final String EXT_ATTRIBUTE_NOTATION = "notation"; // $NON-NLS-N$
```

Comment:
Please reuse constants from the `RegistryConstants` class.

Reply:
What idea behind the RegistryConstants?  all words for all times?

Reply:
It's just a place for all common constants used in extension points. It keeps the code clean by limiting the number of duplicated constants scattered across the code base.